src/backend/marsha/core/views.py

```diff
@@ -101,6 +102,9 @@ def get_context_data(self):
             VideoSerializer(video).data if video else None
         )
 
+        if getattr(settings, "STATICFILES_AWS_ENABLED", False):
+            context["cloudfront_domain"] = settings.CLOUDFRONT_DOMAIN
```

Comment:
I think we need a test for this that checks the rendered html with several settings values (using `override_settings` decorator in tests?).